manifests/overlays/rhoai/params.env

```diff
@@ -0,0 +1 @@
+odh-dashboard-image=quay.io/opendatahub/odh-dashboard:main
```

Comment:
I don't understand what this file is for... this is the upstream ODH image 🤔

Reply:
Yes, let me add something here

Reply:
yes, this is only for rhoai and disconnect env.